### PR TITLE
Fix missing documentation

### PR DESCRIPTION
**Pull request by coryan**
Fix a lot of documentation nits.  This is part of the fixes for #342.

### CODE REVIEW

File: bigtable/client/admin_client.h [hunk 1]
```diff
@@ -24,6 +24,17 @@
...
 inline namespace BIGTABLE_CLIENT_NS {
+/**
+ * Connects to Cloud Bigtable's table administration APIs.
+ *
+ * This class is used by the Cloud Bigtable wrappers to access Cloud Bigtable.
+ * Multiple `bigtable::TableAdmin` objects may share a connection via a
+ * single `AdminClient` object. The `AdminClient` object is configured at
+ * construction time, this configuration includes the credentials, access
+ * endpoints, default timeouts, and other gRPC configuration options. This is an
+ * interface class because it is also used as a dependency injection points in
```
Comment (deepankarsharma) [inline bigtable/client/admin_client.h]: points should probably be point

File: bigtable/client/data_client.h [hunk 2]
```diff
@@ -21,11 +21,15 @@
...
+ * Connects to Cloud Bigtable's data manipulation APIs.
  *
  * This class is used by the Cloud Bigtable wrappers to access Cloud Bigtable.
- * It provides a configuration point to control how we do load-balancing, how we
- * handle reconnections, refresh authorization tokens, etc.
+ * Multiple `bigtable::Table` objects may share a connection via a single
+ * `DataClient` object. The `DataClient` object is configured at construction
+ * time, this configuration includes the credentials, access endpoints, default
+ * timeouts, and other gRPC configuration options. This is an interface class
+ * because it is also used as a dependency injection points in some of the
```
Comment (deepankarsharma) [inline bigtable/client/data_client.h]: points should probably be point

File: bigtable/client/instance_admin_client.h [hunk 3]
```diff
@@ -22,6 +22,17 @@
...
+/**
+ * Connects to Cloud Bigtable's instance administration APIs.
+ *
+ * This class is used by the Cloud Bigtable wrappers to access Cloud Bigtable.
+ * Multiple `bigtable::InstanceAdmin` objects may share a connection via a
+ * single `InstanceAdminClient` object. The `InstanceAdminClient` object is
+ * configured at construction time, this configuration includes the credentials,
+ * access endpoints, default timeouts, and other gRPC configuration options.
+ * This is an interface class because it is also used as a dependency injection
+ * points in some of the tests.
```
Comment (deepankarsharma) [inline bigtable/client/instance_admin_client.h]: points should probably be point

File: bigtable/client/admin_client.h (hunk 1 above)
Comment (coryan) [reply bigtable/client/admin_client.h]: Fixed.

File: bigtable/client/data_client.h (hunk 2 above)
Comment (coryan) [reply bigtable/client/data_client.h]: Fixed.

File: bigtable/client/instance_admin_client.h (hunk 3 above)
Comment (coryan) [reply bigtable/client/instance_admin_client.h]: Fixed.

**Comment by coryan** [comment]
PTAL.

**Pull request merged**